### DISCUSSION TITLE
[Feature] Promote user to admin

### DISCUSSION THREAD

**Discussion by indrora** [immich-app/immich#3295] [Feature Request]
### The feature

It would be nice to be able to promote a user to administrator, that way multiple users could manage jobs/etc. 

I generally have an "Administrative" account that I don't log into and a "User" account that I use on a regular basis; being able to elevate a user to do administrative tasks would be useful.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by jpb0418** [comment]
if you exec in to the db container and run psql immich postgres and UPDATE users SET "isAdmin" = 't' WHERE email = 'EMAILOFUSER';

you can run SELECT * FROM users; to see the users

**Reply by edmondsiu0** [reply]
With a bit of formatting:

### Connect to the DB
(run from within shell in the postgres container)
```
psql immich postgres
```

### List all users
```
SELECT * FROM users;
```

### To promote an existing user
```
UPDATE users SET "isAdmin"='t' WHERE email='email@example.com';
```

**Reply by fraankkr** [reply]
I'm new to the docker world. how can i exec into the postgres container. using portainer for example gives me an error when trying to open the shell there. Thanks.

**Reply by nanjarapalli** [reply]
```
# CONTAINERID  refers to postgress container tensorchord/pgvecto-rs:pg14-v0.2.0
docker exec -it CONTAINERID bin/bash
psql immich postgres
SELECT * FROM users;
UPDATE users SET "isAdmin"='t' WHERE email='email@example.com';
```

**Comment by daschillinger** [comment]
Would be nice to do it in the webinterface

**Comment by ravensorb** [comment]
Would love to see this feature added!

**Comment by bo0tzz** [comment]
Added in #15380
